results/intfloat__multilingual-e5-small/MAUDGeneralEconomicAndFinancialConditionsSubjectToDisproportionateImpactModifierLegalBenchClassification.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dataset_revision": "12ca3b695563788fead87a982ad1a068284413f4",
+  "mteb_dataset_name": "MAUDGeneralEconomicAndFinancialConditionsSubjectToDisproportionateImpactModifierLegalBenchClassification",
```

Comment:
Long name like these will probably cause issues for benchmark viz. later on. It might be worth reducing the size.

Reply:
Maybe we can remove some context to shorten it?

`MAUDGeneralEconomicAndFinancialConditionsSubjectToDisproportionateImpactModifierLegalBenchClassification` to `MAUDEconomicConditionsDisproportionateImpactLegalBenchClassification` 

The meta data would have more details and citation.